### ISSUE TITLE
Return to redial after exit ssh session

### ISSUE THREAD

**Issue by telefax** [taypo/redial#46]
Would it be possible to have redial launch again after you have exited your ssh session?
You already have a state file so it would be pretty nice to return to where you left last time. I think this is a pretty expected behaviour when you work at scale.

Thanks :)

**Comment by taypo** [comment]
Yeah, we already do that. Connect to some server and exit immidiaetly, you return to redial. But if you hit ctrl+c during your session, redial also exits. See #28

**Comment by telefax** [comment]
Wow, it works now. For some reason, I hit exit earlier and just got back to shell. But now when I try it works. Odd. Now I can only reproduce this if I enter wrong password or dont supply ssh key, I get thrown back to shell.
